### ISSUE TITLE
dev --watch breaks on second run

### ISSUE THREAD

**Issue by acrobat** [symfony/webpack-encore#121]
When I run the `./node_modules/.bin/encore dev --watch` command it compiles the assets but after another change I get an exception. Using version 0.11.0 of webpack-encore

Output:
```
→ ./node_modules/.bin/encore dev --watch
Running webpack ...


Webpack is watching the files…

(node:15488) DeprecationWarning: Chunk.modules is deprecated. Use Chunk.getNumberOfModules/mapModules/forEachModule/containsModule instead.
 DONE  Compiled successfully in 8405ms                                                                                                      6:08:37 PM

 I  50 files written to web/build
 WAIT  Compiling...                                                                                                                         6:08:47 PM

/path/to/project/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:145
			if(key.startsWith(what))
			      ^

TypeError: Cannot read property 'startsWith' of undefined
    at Storage.purge (/path/to/project/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:145:10)
    at Storage.purge (/path/to/project/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:150:9)
    at CachedInputFileSystem.purge (/path/to/project/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:259:20)
    at EventEmitter.watcher.once (/path/to/project/node_modules/webpack/lib/node/NodeWatchFileSystem.js:42:26)
    at EventEmitter.g (events.js:292:16)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:191:7)
    at EventEmitter._onTimeout (/path/to/project/node_modules/watchpack/lib/watchpack.js:142:7)
    at ontimeout (timers.js:386:14)
    at tryOnTimeout (timers.js:250:5)
    at Timer.listOnTimeout (timers.js:214:5)
```

-----------------------------------------------

With webpack-encore 0.12.0 I get this error:
```
→ ./node_modules/.bin/encore dev --watch
Running webpack ...


Webpack is watching the files…

(node:16147) DeprecationWarning: Chunk.modules is deprecated. Use Chunk.getNumberOfModules/mapModules/forEachModule/containsModule instead.
 ERROR  Failed to compile with 1 errors                                                                                                     6:16:13 PM

 error  

Entry module not found: Error: Can't resolve 'dev' in '/path/to/project'
```

**Comment by Lyrkan** [comment]
Hi @acrobat,

About your first issue it seems that it is caused by [this bug](https://github.com/webpack/enhanced-resolve/issues/97) of `enhanced-resolve`. If you run ` yarn list` you'll probably see the version 3.3.0 being used by Webpack, which is a depency of Encore.

Since it has been resolved with the version 3.3.1 of `enhanced-resolve`, you can do one the two following things to update it:

* `yarn upgrade`: be careful, this will update all the depdendencies of your project
* `yarn remove @symfony/webpack-encore && yarn add @symfony/webpack-encore --dev`: this will only update dependencies related to Encore. The `remove` part is needed to be sure that Yarn will try to update sub-dependencies (if you only do `yarn upgrade @symfony/webpack-encore`, the version of `enhanced-resolve` will probably stay the same).

About the second issue... that's odd, it is supposed to be fixed in 0.12.0 (see #112).

**Comment by acrobat** [comment]
Thanks @Lyrkan removing and adding webpack-encore did solve my issue!
